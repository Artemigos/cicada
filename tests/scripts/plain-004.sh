touch 'foo.1' 'bar.1' 'foo.txt'
ls *.1 | sort

ls f* | sort

echo 'f*'
echo "bar*"

rm -f 'foo.1' 'bar.1' 'foo.txt'
