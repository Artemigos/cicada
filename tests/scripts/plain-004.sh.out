bar.1
foo.1
foo.1
foo.txt
f*
bar*
